### PR TITLE
Design for main page

### PR DESCRIPTION
**Pull request by 47souls**
Hey guys I`ve done some primary design for our main page, so now it contains header and sliding menu. Our components will be shown in content div.

### CODE REVIEW

**Comment by KostyaTk** [comment]
reviewed

**Comment by Sabfir** [comment]
reviewed

File: src/main/webapp/WEB-INF/views/main.jsp [hunk 1]
```diff
@@ -3,25 +3,72 @@
...
-
-	<br>
-	<br>
-	<h4>Old jsp code:</h4>
+	<header>
+        <nav role="navigation">
+	       <div class="nav-wrapper">
+                <div class="row">
+                    <div class="col s2 m1 l1">
+                        <i id="toggle-button" class="circle waves-effect waves-light material-icons" style="width:40px; height:40px; margin-left: 20px">menu</i>
```
Comment (Michaluch) [inline src/main/webapp/WEB-INF/views/main.jsp]: This is bad solution. Move style from "style" atribute into css file. Avoid similar situations in the future, please.

File: src/main/webapp/WEB-INF/views/main.jsp [hunk 2]
```diff
@@ -3,25 +3,72 @@
...
+                            <a href="#/profile" class="collection-item"><i class="material-icons">perm_identity</i>Profile</a>
+                        </div>
+                        <div class="content">
+                            <ng-view></ng-view>
+                        </div>
+            </div>   
+    </main>
+    
+	<!-- <h4>Old jsp code:</h4>
 	<c:choose>
```
Comment (Michaluch) [inline src/main/webapp/WEB-INF/views/main.jsp]: Drop unused code.

**Comment by Michaluch** [comment]
Why you use two different files for nav-bar deffinition? Join content.css and nav-bar.css into one file.

File: src/main/webapp/WEB-INF/views/main.jsp (hunk 1 above)
Comment (47souls) [inline src/main/webapp/WEB-INF/views/main.jsp]: Well, ok) I`ll avoid this in future, in this situation used inline style just for testing and forgot to move it into css file, sorry, my fault)

File: src/main/webapp/WEB-INF/views/main.jsp (hunk 2 above)
Comment (47souls) [inline src/main/webapp/WEB-INF/views/main.jsp]: Droped

**Pull request merged**